### PR TITLE
Aswathy/ Refactored the script for CIO

### PR DESCRIPTION
**Pull request by aswathy-deriv**
Changes:

-  Refactored the CIO addscript in the academy

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [x] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/academy/components/utility.ts [hunk 1]
```diff
@@ -1,3 +1,42 @@
...
+    const addScript = (settings) => {
+        const script = document.createElement('script')
+        const { async, text, src, id } = settings
+
+        if (async) script.async = settings['async']
+        if (text) script.text = settings['text']
+        if (src) script.src = settings['src']
+        if (id) script.id = settings['id']
+        document.body.appendChild(script)
+    }
```
Comment (prince-deriv) [inline src/pages/academy/components/utility.ts:25]: This should use the existing `addScript` on `src/common/utility.js`

Comment (aswathy-deriv) [reply src/pages/academy/components/utility.ts:25]: okay @prince-deriv

**Review by prince-deriv — DISMISSED** [review]

**Review by NikitaK-deriv — APPROVED** [review]
LGTM :+1:

**Review by vijayasree-deriv — DISMISSED** [review]

**Review by mohammad-rework — DISMISSED** [review]

**Review by prince-deriv — DISMISSED** [review]

**Review by sean-binary — APPROVED** [review]

**Review by sean-binary — DISMISSED** [review]
LGTM

**Review by prince-deriv — DISMISSED** [review]

**Review by prince-deriv — APPROVED** [review]

**Review by prince-deriv — APPROVED** [review]

**Pull request merged**